tendrl/node_agent/discovery_modules/plugin.py

```diff
@@ -0,0 +1,11 @@
+class PluginMount(type):
```

Comment:
See the comment https://github.com/Tendrl/performance-monitoring/pull/23/commits/5a32480de1d7d369461601337683439803914255#r95658651
So it might pointless to keep it in 'commons'. So probably we can keep one for discovery plugins as long as it holds good